### ISSUE TITLE
Cannot see mpv UI anymore

### ISSUE THREAD

**Issue by LeKite** [mpv-player/mpv#13930]
### Important Information

Provide following Information:
- mpv version: pacman -Qi package-name
- Linux Distribution and Version: Arch Stable
- Source of the mpv binary
- If known which version of mpv introduced the problem: current one
- Window Manager and version: KDE Plasma 6
- GPU model, driver and version: 2060s nvidia driver
- Possible screenshot or video of visual glitches

### Reproduction steps

Open a video.

### Expected behavior
Being able to see the UI, to browse the video etc..

### Actual behavior
I cannot see anything.
### Log file
[output.txt](https://github.com/mpv-player/mpv/files/15030162/output.txt)

Any help is appreciated.

**Comment by guidocella** [comment]
`osc=no` in `mpv.conf` disables the GUI.

**Comment by CounterPillow** [comment]
This is a problem in one of your many third party scripts

**Comment by LeKite** [comment]
> `osc=no` in `mpv.conf` disables the GUI.
Mm yes but I had it till yesterday working. I think because I use modernx.lua. Today doesn't work anymore.



> This is a problem in one of your many third party scripts

So I should go to their github page you say?

Anyway I attached more info

**Comment by LeKite** [comment]
> > `osc=no` in `mpv.conf` disables the GUI.

Mm yes but I had it till yesterday working. I think because I use modernx.lua. Today doesn't work anymore.

> > This is a problem in one of your many third party scripts

So I should go to their github page you say?

Anyway I attached more info

**Comment by CounterPillow** [comment]
> So I should go to their github page you say?

Yes you should not report third party UI scripts being broken to upstream mpv unless it is determined to be a bug in mpv.

**Comment by LeKite** [comment]
I'm sorry if I made a mistake. I was not sure where the issue was.
